helm/examples/azure/values-retained-azurefile.yaml

```diff
@@ -36,7 +36,11 @@ storage:
     labels:
     - data # NB: if not set, this would be the default
 
-
+  - name: dropzone
+    pvc: dropzone-azstorage-hpcc-azurefile-pvc
+    prefix: "/var/lib/HPCCSystems/dropzone"
+    labels:
+    - lz # NB: if not set, this would be the default
```

Comment:
I think this will make it the only example to have a LZ.
That's okay, but once HPCC-25952 is merged, we should make all examples consistent, and add a default LZ to them too.
Can you open a JIRA so it is remembered, and link it to HPCC-25952 ?

Reply:
https://track.hpccsystems.com/browse/HPCC-25983 which reference HPCC-25952